src/averageAges.js

```diff
@@ -36,10 +41,21 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(({ sex }) => sex === 'f');
+
+  const mothers = women.filter(({ name }) =>
+    people.some(({ mother }) => mother === name));
```

Comment:
```suggestion
  const mothers = women.filter(({ name }) =>
    people.some(({ mother }) => mother === name),
  );
```

Reply:
Ok, but why? I should always close functions like this at a new line?

Reply:
Ok, but why? I should always close functions like this at a new line?